app/presenters/project_show_presenter.rb

```diff
@@ -100,4 +108,14 @@ def storage_remaining(session_id:)
       remaining = (capacity/default_capacity_divisor) - usage
       remaining*default_capacity_divisor
     end
+
+    def xml_presenter_args
+      [
```

Comment:
I think utilizing the array and then the splat overcomplicates this passing.  I'm unsure why you are not just passing the project directly to the presenter, but if this indirection is needed then please make it a bit simpler.
```suggestion
```